Skate_ERP_preprocessing_IO.m

```diff
@@ -1,39 +1,30 @@
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%                    REGULAR IN
-%
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%%
 clear all
 close all
 ccc
 
 exp = 'Skateboard';
-
-% loading regulars first
-subs = {'100' '101' '102' '103' '104' '106' '108' '111'...
-  '112' '116' '117' '118' '119' '120'};
+subs = {'100' '101' '102' '103' '104' '106' '107' '108' '109' '110' '111'...
+    '112' '113' '114' '115' '116' '117' '118' '119' '120' '122'};
+is_goofy = [0, 0, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1];
 %subs = {'100'}; %to test on just one sub
 
- nsubs = length(subs);
-%conds = {'P_CW';'P_CCW'; 'NP_CW'; 'NP_CCW'};
-conds =  {'P_CW';'NP_CCW'}; %facing inside for regulars
-nconds = length(conds);
+nsubs = length(subs);
+conds = {'P_CW';'P_CCW'; 'NP_CW'; 'NP_CCW'};
+new_conds = {'Facing_In'; 'Facing_Out'};
+n_new = length(new_conds);
 Pathname = 'M:\Data\Skateboard\winter2019\';
 
 if exist([Pathname 'segments\'])
-    mkdir([Pathname 'segmentsIO\']);
+    mkdir([Pathname 'segments_IO_V2\']);
 end
 [ALLEEG EEG CURRENTSET ALLCOM] = eeglab;
 
-
-for i_sub = 1:nsubs
-    for i_cond = 1:nconds
-
-                        
+for i_sub = 1:length(subs)
```

Comment:
these should be nconds and nsubs not sure why you changed them

Reply:
I was following the example of the code.  It's easy to set it up the normal way we do it. That's what all the other codes are like. easy change.